src/quo/components/wallet/wallet_activity/view.cljs

```diff
@@ -118,10 +118,10 @@
       :on-press-out        on-press-out}
      [rn/view
       {:style {:flex-direction :row}}
-      [transaction-icon-view props]
+      [transaction-icon-view props theme]
       [rn/view
        {:style style/content-container}
-       [transaction-header props]
+       [transaction-header props theme]
```

Comment:
Is there any reasoning for moving themes outside of props map?

Reply:
@shivekkhurana we no longer use the old theme approach where it was in the props. We now get theme from 
`theme         (quo.theme/use-theme)`